newIDE/app/src/ObjectsRendering/Renderers/Rendered3DInstance.js

```diff
@@ -87,17 +87,31 @@ export default class Rendered3DInstance {
   }
 
   getCenterX() {
-    if (this._instance.hasCustomSize())
-      return this._instance.getCustomWidth() / 2;
-
-    return this.getDefaultWidth() / 2;
+    return this.getWidth() / 2;
   }
 
   getCenterY() {
-    if (this._instance.hasCustomSize())
-      return this._instance.getCustomHeight() / 2;
+    return this.getHeight() / 2;
+  }
+
+  getCustomWidth(): number {
```

Comment:
what's the point of the functions `getCustomWidth` and `getCustomHeight` if they return the same thing as `getWidth()` and `getHeight()`?

Reply:
(because if instance `hasCustomSize` is true, then it should have a `customWidth` and vice versa

Reply:
It's not the same thing.
- `getCustomWidth` returns the default width if no custom width has ever been set by users.
- `getWidth()` returns the default width if user uncheck the box "Custom size".

Reply:
ok, I find the UX a bit confusing with this checkbox but maybe that's just me

Reply:
I could be a reset to default button, but it would only be a way.